app/src/main/java/eu/kanade/tachiyomi/data/download/DownloadNotifier.kt

```diff
@@ -198,6 +214,14 @@ internal class DownloadNotifier(private val context: Context) {
             setSmallIcon(R.drawable.ic_warning_white_24dp)
             clearActions()
             setContentIntent(NotificationHandler.openDownloadManagerPendingActivity(context))
+            if (mangaId != null) {
+                addAction(
+                    R.drawable.ic_book_24dp,
+                    context.stringResource(MR.strings.action_show_manga),
+
```

Comment:
```suggestion
```

Reply:
These suggestions are only removing blank lines

Reply:
> Ah ok. It is very hard to tell in both the "review files" and the "conversations" interface.

![grafik](https://github.com/user-attachments/assets/53a320e8-b462-4be4-8e33-46aedbaffe53)
this is the box for the actual change, the code bit above it is the context for the suggestion